include/SFML/Graphics/Color.inl

```diff
@@ -107,8 +107,7 @@ constexpr Color operator*(const Color& left, const Color& right)
 {
     const auto scaledMul = [](std::uint8_t lhs, std::uint8_t rhs) -> std::uint8_t
     {
-        const auto uint16Result = static_cast<std::uint16_t>(
-            static_cast<std::uint16_t>(lhs) * static_cast<std::uint16_t>(rhs));
+        const auto uint16Result = static_cast<std::uint16_t>(std::uint16_t{lhs} * std::uint16_t{rhs});
```

Comment:
What's the reason for leaving one static cast here? could it not be braces init like the others on this line?

Reply:
I use braces when we need to do a promotion (`uint8_t` -> `uint16_t` loses no data so braces can be used) and `static_cast` when we need to discard information. Multiplying two `uint16_t`s produces an `int` which must be `static_cast`'ed down back to a `uint16_t`.